mainframer

```diff
@@ -115,8 +120,18 @@ function executeRemoteCommand {
 	startTime="$(date +%s)"
 
 	set +e
-	if ssh "$REMOTE_MACHINE" "echo 'set -e && cd '$PROJECT_DIR_ON_REMOTE_MACHINE' && echo \"$REMOTE_COMMAND\" && echo "" && $REMOTE_COMMAND' | bash" ; then
-		REMOTE_COMMAND_SUCCESSFUL="true"
+	if [ "$LOCAL_DIR_IN_OUTPUT" == "true" ]; then
+		REMOTE_PROJECT_DIR_ABSOLUTE=$(ssh "$REMOTE_MACHINE" "cd $PROJECT_DIR_ON_REMOTE_MACHINE && pwd")
+
+		if ssh "$REMOTE_MACHINE" "echo 'set -e && cd '$PROJECT_DIR_ON_REMOTE_MACHINE' && echo \"$REMOTE_COMMAND\" && echo "" && $REMOTE_COMMAND' | bash" \
+		2> >(sed -l "s|$REMOTE_PROJECT_DIR_ABSOLUTE|$PROJECT_DIR|g" >&2) \
+		1> >(sed -l "s|$REMOTE_PROJECT_DIR_ABSOLUTE|$PROJECT_DIR|g" >&1); then
```

Comment:
-l is important to process the output line-by-line instead of waiting for the remote build to complete